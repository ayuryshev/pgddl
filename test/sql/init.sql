CREATE EXTENSION ddlx;
\pset tuples_only

\dx ddlx

