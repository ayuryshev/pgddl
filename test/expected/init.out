CREATE EXTENSION ddl;
\dx ddl
           List of installed extensions
 Name | Version | Schema |       Description       
------+---------+--------+-------------------------
 ddl  | 0.1     | public | DDL extractor functions
(1 row)

\dx+ ddl
           Objects in extension "ddl"
               Object Description               
------------------------------------------------
 function pg_ddl_alter_owner(oid)
 function pg_ddl_alter_table_defaults(regclass)
 function pg_ddl_banner(text,text,text,text)
 function pg_ddl_comment(oid)
 function pg_ddl_create_class(regclass)
 function pg_ddl_create_constraints(regclass)
 function pg_ddl_create_function(regproc)
 function pg_ddl_create_indexes(regclass)
 function pg_ddl_create_rules(regclass)
 function pg_ddl_create_table(regclass)
 function pg_ddl_create_triggers(regclass)
 function pg_ddl_create_type_domain(regtype)
 function pg_ddl_create_type_enum(regtype)
 function pg_ddl_create_view(regclass)
 function pg_ddl_get_columns(regclass)
 function pg_ddl_get_constraints(regclass)
 function pg_ddl_get_functions(regproc)
 function pg_ddl_get_indexes(regclass)
 function pg_ddl_get_rules(regclass)
 function pg_ddl_get_triggers(regclass)
 function pg_ddl_grants_on_class(regclass)
 function pg_ddl_grants_on_proc(regproc)
 function pg_ddl_oid_info(oid)
 function pg_ddl_script(regclass)
 function pg_ddl_script(regproc)
 function pg_ddl_script(regprocedure)
 function pg_ddl_script(regtype)
(27 rows)

